smother/control.py

```diff
@@ -81,6 +86,13 @@ def write(self, file_or_path, append=False, timeout=10):
         multiprocess test environment.
         """
         if isinstance(file_or_path, six.string_types):
+            if self.coverage and self.coverage.config.parallel:
```

Comment:
Ok that SGTM -- let's just update the docstring to note when and how file_or_path is renamed